README.md

```diff
@@ -1,24 +1,170 @@
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<!-- Features -->
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- Handles logic for storing and retrieving data from the database
+- Handles logic for returning data to the frontend
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎨 Link to Frontend <a name="key-features"></a>
+
+- [Frontend](https://github.com/ginabeki/hello-react-front-end/tree/create-messages)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ git
+ bundle
+ Your favorite text editor
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/ginabeki/hello-rails-back-end.git
+```
+
+### Database setup
+
+Connect to your local postgres database:
+
+### Install
+
+Install this project with:
+
+```sh
+  cd hello-rails-back-end
+  bundle install
+  rails: db:create - create the database
+  rails: db:migrate - run pending migrations for the current environment.
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server - Starts the development server
+  rails console - Starts the interactive command line
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec - Runs the test suite
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Gina Beki**
+
+- GitHub: [@ginabeki](https://github.com/ginabeki)
+- LinkedIn: [Gina Beki](https://www.linkedin.com/in/ginabeki)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] Add more endpoints
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please leave a star🤩
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [Required] Please consider adding a `License` file to the root directory of your project and also a `link` to the license on the `readme` file. Currently, when I click on this link I get a `404` page not found error. Kindly add a License to the project, this will inform someone who wants to use your project how it is Licensed.